release.config.js

```diff
@@ -63,5 +63,6 @@ module.exports = {
 };
 
 function equalsIgnoreCase(a, b) {
+  if (!a || !b) return false
```

Comment:
This could return `false` if `a == b && a == null`, should it return true if `null == null`?